teamcity: failed test: TestImportMysql [skipped]

The following tests appear to have failed on master (testrace): TestImportMysql/single_table_dump, TestImportMysql/second_table_dump, TestImportMysql/second_from_multi, TestImportMysql, TestImportMysql/simple_from_multi, TestImportMysql/all_from_multi, TestImportMysql/read_data_only, TestImportMysql/all_from_multi_gzip

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestImportMysql).

[#1456894](https://teamcity.cockroachdb.com/viewLog.html?buildId=1456894):


```
TestImportMysql/read_data_only
--- FAIL: testrace/TestImportMysql/read_data_only (0.000s)
Test ended in panic.

------- Stdout: -------
I190827 19:40:45.978158 137211 server/status/runtime.go:498  [n1] runtime stats: 2.0 GiB RSS, 704 goroutines, 128 MiB/153 MiB/330 MiB GO alloc/idle/total, 86 MiB/134 MiB CGO alloc/total, 2659.8 CGO/sec, 130.2/11.4 %(u/s)time, 0.2 %gc (3x), 3.5 MiB/3.5 MiB (r/w)net
I190827 19:40:46.010979 140291 storage/replica_command.go:293  [n1,s1,r20/1:/{Table/24-Max}] initiating a split of this range at key /Table/53/1 [r21] (manual)
I190827 19:40:46.644352 140438 storage/replica_command.go:293  [n1,split,s1,r20/1:/Table/{24-53/1}] initiating a split of this range at key /Table/53 [r22] (zone config)



TestImportMysql
...storage.go:814  [n3,s3,r1/3:/{Min-System/NodeL…}] applied LEARNER snapshot [total=12ms ingestion=4@5ms id=1d7eca90 index=229]
I190827 19:40:43.562945 137028 storage/replica_command.go:1419  [n1,replicate,s1,r1/1:/{Min-System/NodeL…}] change replicas (add [(n3,s3):3] remove []): existing descriptor r1:/{Min-System/NodeLiveness} [(n1,s1):1, (n2,s2):2, (n3,s3):3LEARNER, next=4, gen=3]
I190827 19:40:43.693562 137028 storage/replica_raft.go:291  [n1,s1,r1/1:/{Min-System/NodeL…}] proposing ADD_REPLICA[(n3,s3):3]: after=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I190827 19:40:43.761193 137028 storage/replica_command.go:1419  [n1,replicate,s1,r17/1:/Table/2{1-2}] change replicas (add [(n3,s3):2LEARNER] remove []): existing descriptor r17:/Table/2{1-2} [(n1,s1):1, next=2, gen=0]
I190827 19:40:43.859628 137028 storage/replica_raft.go:291  [n1,s1,r17/1:/Table/2{1-2}] proposing ADD_REPLICA[(n3,s3):2LEARNER]: after=[(n1,s1):1 (n3,s3):2LEARNER] next=3
I190827 19:40:43.935099 137028 storage/store_snapshot.go:995  [n1,replicate,s1,r17/1:/Table/2{1-2}] sending LEARNER snapshot b46cf9f9 at applied index 15
I190827 19:40:43.944239 137028 storage/store_snapshot.go:1038  [n1,replicate,s1,r17/1:/Table/2{1-2}] streamed snapshot to (n3,s3):2LEARNER: kv pairs: 6, log entries: 0, rate-limit: 8.0 MiB/sec, 0.01s
I190827 19:40:43.947934 139886 storage/replica_raftstorage.go:793  [n3,s3,r17/2:{-}] applying LEARNER snapshot [id=b46cf9f9 index=15]
I190827 19:40:43.973964 139886 storage/replica_raftstorage.go:814  [n3,s3,r17/2:/Table/2{1-2}] applied LEARNER snapshot [total=12ms ingestion=4@2ms id=b46cf9f9 index=15]
I190827 19:40:44.010072 137028 storage/replica_command.go:1419  [n1,replicate,s1,r17/1:/Table/2{1-2}] change replicas (add [(n3,s3):2] remove []): existing descriptor r17:/Table/2{1-2} [(n1,s1):1, (n3,s3):2LEARNER, next=3, gen=1]
I190827 19:40:44.070328 137028 storage/replica_raft.go:291  [n1,s1,r17/1:/Table/2{1-2}] proposing ADD_REPLICA[(n3,s3):2]: after=[(n1,s1):1 (n3,s3):2] next=3
I190827 19:40:44.076374 137028 storage/replica_command.go:1419  [n1,replicate,s1,r17/1:/Table/2{1-2}] change replicas (add [(n2,s2):3LEARNER] remove []): existing descriptor r17:/Table/2{1-2} [(n1,s1):1, (n3,s3):2, next=3, gen=2]
I190827 19:40:44.229710 137028 storage/replica_raft.go:291  [n1,s1,r17/1:/Table/2{1-2}] proposing ADD_REPLICA[(n2,s2):3LEARNER]: after=[(n1,s1):1 (n3,s3):2 (n2,s2):3LEARNER] next=4
I190827 19:40:44.284920 137028 storage/store_snapshot.go:995  [n1,replicate,s1,r17/1:/Table/2{1-2}] sending LEARNER snapshot 3c9865b0 at applied index 21
I190827 19:40:44.286083 137028 storage/store_snapshot.go:1038  [n1,replicate,s1,r17/1:/Table/2{1-2}] streamed snapshot to (n2,s2):3LEARNER: kv pairs: 11, log entries: 0, rate-limit: 8.0 MiB/sec, 0.01s
I190827 19:40:44.299800 139990 storage/replica_raftstorage.go:793  [n2,s2,r17/3:{-}] applying LEARNER snapshot [id=3c9865b0 index=21]
I190827 19:40:44.308624 139990 storage/replica_raftstorage.go:814  [n2,s2,r17/3:/Table/2{1-2}] applied LEARNER snapshot [total=8ms ingestion=4@1ms id=3c9865b0 index=21]
I190827 19:40:44.325211 137028 storage/replica_command.go:1419  [n1,replicate,s1,r17/1:/Table/2{1-2}] change replicas (add [(n2,s2):3] remove []): existing descriptor r17:/Table/2{1-2} [(n1,s1):1, (n3,s3):2, (n2,s2):3LEARNER, next=4, gen=3]
I190827 19:40:44.409266 137028 storage/replica_raft.go:291  [n1,s1,r17/1:/Table/2{1-2}] proposing ADD_REPLICA[(n2,s2):3]: after=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I190827 19:40:44.480819 137028 testutils/testcluster/testcluster.go:656  WaitForFullReplication took: 13.747788778s
I190827 19:40:44.844635 139891 sql/event_log.go:130  [n1,client=127.0.0.1:49848,user=root] Event: "set_cluster_setting", target: 0, info: {SettingName:kv.bulk_ingest.batch_size Value:10KB User:root}
I190827 19:40:45.209642 139891 sql/event_log.go:130  [n1,client=127.0.0.1:49848,user=root] Event: "create_database", target: 52, info: {DatabaseName:foo Statement:CREATE DATABASE foo User:root}



TestImportMysql/simple_from_multi
...ica r28/2
I190827 19:41:03.088963 138025 storage/compactor/compactor.go:325  [n2,s2,compactor] purging suggested compaction for range /Table/53/1 - /Max that contains live data
I190827 19:41:03.089355 138025 storage/compactor/compactor.go:325  [n2,s2,compactor] purging suggested compaction for range /Table/54/1 - /Table/55 that contains live data
I190827 19:41:03.089561 138025 storage/compactor/compactor.go:325  [n2,s2,compactor] purging suggested compaction for range /Table/55/1 - /Max that contains live data
I190827 19:41:03.089723 138025 storage/compactor/compactor.go:325  [n2,s2,compactor] purging suggested compaction for range /Table/56/1 - /Table/56/2 that contains live data
I190827 19:41:03.089850 138025 storage/compactor/compactor.go:325  [n2,s2,compactor] purging suggested compaction for range /Table/56/2 - /Max that contains live data
I190827 19:41:03.090027 138025 storage/compactor/compactor.go:325  [n2,s2,compactor] purging suggested compaction for range /Table/56/3 - /Max that contains live data
I190827 19:41:03.090875 137205 storage/compactor/compactor.go:325  [n1,s1,compactor] purging suggested compaction for range /Table/53/1 - /Max that contains live data
I190827 19:41:03.091141 137205 storage/compactor/compactor.go:325  [n1,s1,compactor] purging suggested compaction for range /Table/54/1 - /Table/55 that contains live data
I190827 19:41:03.091292 137205 storage/compactor/compactor.go:325  [n1,s1,compactor] purging suggested compaction for range /Table/55/1 - /Max that contains live data
I190827 19:41:03.091488 137205 storage/compactor/compactor.go:325  [n1,s1,compactor] purging suggested compaction for range /Table/56/1 - /Table/56/2 that contains live data
I190827 19:41:03.091660 137205 storage/compactor/compactor.go:325  [n1,s1,compactor] purging suggested compaction for range /Table/56/2 - /Max that contains live data
I190827 19:41:03.091811 137205 storage/compactor/compactor.go:325  [n1,s1,compactor] purging suggested compaction for range /Table/56/3 - /Max that contains live data
I190827 19:41:03.100482 138338 storage/compactor/compactor.go:325  [n3,s3,compactor] purging suggested compaction for range /Table/53/1 - /Max that contains live data
I190827 19:41:03.100947 138338 storage/compactor/compactor.go:325  [n3,s3,compactor] purging suggested compaction for range /Table/54/1 - /Table/55 that contains live data
I190827 19:41:03.101245 138338 storage/compactor/compactor.go:325  [n3,s3,compactor] purging suggested compaction for range /Table/55/1 - /Max that contains live data
I190827 19:41:03.101577 138338 storage/compactor/compactor.go:325  [n3,s3,compactor] purging suggested compaction for range /Table/56/1 - /Table/56/2 that contains live data
I190827 19:41:03.101896 138338 storage/compactor/compactor.go:325  [n3,s3,compactor] purging suggested compaction for range /Table/56/2 - /Max that contains live data
I190827 19:41:03.105778 138338 storage/compactor/compactor.go:325  [n3,s3,compactor] purging suggested compaction for range /Table/56/3 - /Max that contains live data
I190827 19:41:03.719471 143644 storage/replica_command.go:293  [n1,s1,r30/1:/{Table/56-Max}] initiating a split of this range at key /Table/58/1 [r31] (manual)
I190827 19:41:04.408726 143650 storage/replica_command.go:293  [n1,s1,r30/1:/Table/5{6-8/1}] initiating a split of this range at key /Table/57/1 [r32] (manual)
I190827 19:41:05.784850 143992 storage/replica_command.go:293  [n1,split,s1,r32/1:/Table/5{7/1-8/1}] initiating a split of this range at key /Table/58 [r33] (zone config)
I190827 19:41:05.826417 143932 storage/replica_command.go:293  [n1,split,s1,r30/1:/Table/5{6-7/1}] initiating a split of this range at key /Table/57 [r34] (zone config)
I190827 19:41:05.991111 137211 server/status/runtime.go:498  [n1] runtime stats: 2.1 GiB RSS, 740 goroutines, 123 MiB/158 MiB/330 MiB GO alloc/idle/total, 98 MiB/144 MiB CGO alloc/total, 1122.0 CGO/sec, 123.7/11.3 %(u/s)time, 0.7 %gc (3x), 2.4 MiB/2.4 MiB (r/w)net



TestImportMysql/second_table_dump
...d}] rate limited in MaybeAdd (raftlog): context canceled
I190827 19:40:52.153458 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root] publish: descID=54 (simple_auto_inc) version=3 mtime=2019-08-27 19:40:51.878252225 +0000 UTC
I190827 19:40:52.499641 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root] publish: descID=55 (simple) version=3 mtime=2019-08-27 19:40:51.878252225 +0000 UTC
I190827 19:40:52.720594 139891 sql/event_log.go:130  [n1,client=127.0.0.1:49848,user=root] Event: "drop_table", target: 55, info: {TableName:foo.public.simple Statement:DROP TABLE IF EXISTS simple, second, third, everything CASCADE User:root CascadeDroppedViews:[]}
I190827 19:40:52.894671 139891 sql/lease.go:329  [n1,client=127.0.0.1:49848,user=root,scExec] waiting for 1 leases to expire: desc=[{simple_auto_inc 54 2}]
I190827 19:40:53.074738 141614 storage/replica_command.go:608  [n1,merge,s1,r26/1:/Table/55{-/1}] initiating a merge of r23:/{Table/55/1-Max} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=8] into this range (lhs+rhs has (size=0 B+18 KiB qps=0.00+3.34 --> 3.34qps) below threshold (size=18 KiB, qps=3.34))
I190827 19:40:53.148384 141516 sql/lease.go:923  new lease: 54("simple_auto_inc") ver=3:1566935144.059973712,0, refcount=0
I190827 19:40:53.415462 137148 storage/store.go:2593  [n1,s1,r26/1:/Table/55{-/1}] removing replica r23/1
I190827 19:40:53.417291 138284 storage/store.go:2593  [n3,s3,r26/3:/Table/55{-/1}] removing replica r23/3
I190827 19:40:53.492530 137951 storage/store.go:2593  [n2,s2,r26/2:/Table/55{-/1}] removing replica r23/2
I190827 19:40:53.573924 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root,scExec] publish: descID=55 (simple) version=4 mtime=2019-08-27 19:40:53.539314328 +0000 UTC
I190827 19:40:54.468181 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root] publish: descID=54 (simple_auto_inc) version=4 mtime=2019-08-27 19:40:54.258200921 +0000 UTC
I190827 19:40:54.633453 139891 sql/event_log.go:130  [n1,client=127.0.0.1:49848,user=root] Event: "drop_sequence", target: 54, info: {SequenceName:foo.public.simple_auto_inc Statement:DROP SEQUENCE IF EXISTS simple_auto_inc, third_auto_inc User:root}
I190827 19:40:54.752987 142088 storage/replica_command.go:608  [n1,merge,s1,r25/1:/Table/54{-/1}] initiating a merge of r24:/Table/5{4/1-5} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=10] into this range (lhs+rhs has (size=0 B+41 B qps=0.00+2.04 --> 2.04qps) below threshold (size=41 B, qps=2.04))
I190827 19:40:54.871917 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root,scExec] publish: descID=54 (simple_auto_inc) version=5 mtime=2019-08-27 19:40:54.870072051 +0000 UTC
I190827 19:40:55.119515 137133 storage/store.go:2593  [n1,s1,r25/1:/Table/54{-/1}] removing replica r24/1
I190827 19:40:55.157175 138282 storage/store.go:2593  [n3,s3,r25/3:/Table/54{-/1}] removing replica r24/3
I190827 19:40:55.169138 137995 storage/store.go:2593  [n2,s2,r25/2:/Table/54{-/1}] removing replica r24/2
I190827 19:40:55.971641 137211 server/status/runtime.go:498  [n1] runtime stats: 2.0 GiB RSS, 719 goroutines, 186 MiB/101 MiB/330 MiB GO alloc/idle/total, 92 MiB/142 MiB CGO alloc/total, 1324.9 CGO/sec, 124.8/11.0 %(u/s)time, 0.1 %gc (2x), 2.7 MiB/2.7 MiB (r/w)net
I190827 19:40:56.059667 142352 storage/replica_command.go:293  [n1,s1,r26/1:/{Table/55-Max}] initiating a split of this range at key /Table/56/1 [r27] (manual)
I190827 19:40:56.375591 142454 storage/replica_command.go:293  [n1,s1,r27/1:/{Table/56/1-Max}] initiating a split of this range at key /Table/56/2 [r28] (manual)
I190827 19:40:56.808721 142415 storage/replica_command.go:293  [n1,s1,r28/1:/{Table/56/2-Max}] initiating a split of this range at key /Table/56/3 [r29] (manual)
I190827 19:40:58.283246 142695 storage/replica_command.go:293  [n1,split,s1,r26/1:/Table/5{5-6/1}] initiating a split of this range at key /Table/56 [r30] (zone config)



TestImportMysql/all_from_multi
.../1 [r39] (manual)
I190827 19:41:19.508083 146383 storage/replica_command.go:293  [n1,s1,r38/1:/Table/{59-63/1}] initiating a split of this range at key /Table/62/1 [r40] (manual)
I190827 19:41:19.604959 138313 storage/queue.go:518  [n3,s3,r9/2:/Table/1{3-4}] rate limited in MaybeAdd (raftlog): context canceled
I190827 19:41:19.830226 146531 storage/replica_command.go:293  [n1,s1,r39/1:/{Table/63/1-Max}] initiating a split of this range at key /Table/65/1 [r41] (manual)
I190827 19:41:20.394106 138359 server/status/runtime.go:498  [n3] runtime stats: 2.1 GiB RSS, 751 goroutines, 209 MiB/79 MiB/331 MiB GO alloc/idle/total, 104 MiB/153 MiB CGO alloc/total, 1318.1 CGO/sec, 128.3/13.2 %(u/s)time, 0.2 %gc (2x), 2.7 MiB/2.7 MiB (r/w)net
I190827 19:41:20.489048 146580 storage/replica_command.go:293  [n1,s1,r41/1:/{Table/65/1-Max}] initiating a split of this range at key /Table/65/2 [r42] (manual)
I190827 19:41:21.092603 146543 storage/replica_command.go:293  [n1,s1,r42/1:/{Table/65/2-Max}] initiating a split of this range at key /Table/65/3 [r43] (manual)
I190827 19:41:21.850137 146668 storage/replica_command.go:293  [n1,s1,r39/1:/Table/6{3/1-5/1}] initiating a split of this range at key /Table/64/1 [r44] (manual)
I190827 19:41:22.285628 146787 storage/replica_command.go:293  [n1,s1,r38/1:/Table/{59-62/1}] initiating a split of this range at key /Table/60/1 [r45] (manual)
I190827 19:41:22.705976 146848 storage/replica_command.go:293  [n1,s1,r45/1:/Table/6{0/1-2/1}] initiating a split of this range at key /Table/61/1 [r46] (manual)
I190827 19:41:23.476388 146878 storage/replica_command.go:293  [n1,s1,r46/1:/Table/6{1/1-2/1}] initiating a split of this range at key /Table/61/2 [r47] (manual); delayed split for 0.2s to avoid Raft snapshot
I190827 19:41:23.860520 146940 storage/replica_command.go:293  [n1,s1,r47/1:/Table/6{1/2-2/1}] initiating a split of this range at key /Table/61/3 [r48] (manual)
I190827 19:41:25.266470 147126 storage/replica_command.go:293  [n1,split,s1,r38/1:/Table/{59-60/1}] initiating a split of this range at key /Table/60 [r49] (zone config)
I190827 19:41:25.293743 147111 storage/replica_command.go:293  [n1,split,s1,r48/1:/Table/6{1/3-2/1}] initiating a split of this range at key /Table/62 [r50] (zone config)
I190827 19:41:25.477314 147255 storage/replica_command.go:293  [n1,split,s1,r40/1:/Table/6{2/1-3/1}] initiating a split of this range at key /Table/63 [r51] (zone config)
I190827 19:41:25.521356 147114 storage/replica_command.go:293  [n1,split,s1,r45/1:/Table/6{0/1-1/1}] initiating a split of this range at key /Table/61 [r52] (zone config)
I190827 19:41:25.954769 137208 gossip/gossip.go:566  [n1] gossip status (ok, 3 nodes)
gossip client (0/3 cur/max conns)
gossip server (2/3 cur/max conns, infos 667/258 sent/received, bytes 643800B/70206B sent/received)
  2: 127.0.0.1:33653 (57s)
  3: 127.0.0.1:32863 (56s)
gossip connectivity
  n2 [sentinel];
  n2 -> n1; n3 -> n1;
I190827 19:41:25.994318 137211 server/status/runtime.go:498  [n1] runtime stats: 2.1 GiB RSS, 758 goroutines, 152 MiB/132 MiB/331 MiB GO alloc/idle/total, 110 MiB/156 MiB CGO alloc/total, 1584.6 CGO/sec, 136.1/13.0 %(u/s)time, 0.2 %gc (3x), 2.9 MiB/2.9 MiB (r/w)net
I190827 19:41:26.000990 147420 storage/replica_command.go:293  [n1,split,s1,r39/1:/Table/6{3/1-4/1}] initiating a split of this range at key /Table/64 [r53] (zone config)
I190827 19:41:26.009856 147364 storage/replica_command.go:293  [n1,split,s1,r44/1:/Table/6{4/1-5/1}] initiating a split of this range at key /Table/65 [r54] (zone config)
W190827 19:41:26.551291 142428 sql/stats/automatic_stats.go:450  failed to create statistics on table 55: create-stats: table is being dropped
W190827 19:41:26.587462 142428 sql/stats/automatic_stats.go:450  failed to create statistics on table 53: create-stats: table is being dropped
W190827 19:41:26.971729 142428 sql/stats/automatic_stats.go:450  failed to create statistics on table 54: create-stats: table is being dropped



TestImportMysql/single_table_dump
--- FAIL: testrace/TestImportMysql/single_table_dump (0.000s)
Test ended in panic.

------- Stdout: -------
I190827 19:40:47.485098 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root] publish: descID=53 (simple) version=3 mtime=2019-08-27 19:40:46.960755156 +0000 UTC
I190827 19:40:47.697516 139891 sql/event_log.go:130  [n1,client=127.0.0.1:49848,user=root] Event: "drop_table", target: 53, info: {TableName:foo.public.simple Statement:DROP TABLE IF EXISTS simple, second, third, everything CASCADE User:root CascadeDroppedViews:[]}
I190827 19:40:47.827470 140652 storage/replica_command.go:608  [n1,merge,s1,r22/1:/Table/53{-/1}] initiating a merge of r21:/{Table/53/1-Max} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=5] into this range (lhs+rhs has (size=0 B+18 KiB qps=0.00+4.59 --> 4.59qps) below threshold (size=18 KiB, qps=4.59))
I190827 19:40:47.916462 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root,scExec] publish: descID=53 (simple) version=4 mtime=2019-08-27 19:40:47.908646897 +0000 UTC
I190827 19:40:48.082489 137160 storage/store.go:2593  [n1,s1,r22/1:/Table/53{-/1}] removing replica r21/1
I190827 19:40:48.087005 137975 storage/store.go:2593  [n2,s2,r22/2:/Table/53{-/1}] removing replica r21/2
I190827 19:40:48.096544 138301 storage/store.go:2593  [n3,s3,r22/3:/Table/53{-/1}] removing replica r21/3
I190827 19:40:49.227942 138030 server/status/runtime.go:498  [n2] runtime stats: 2.0 GiB RSS, 697 goroutines, 218 MiB/76 MiB/330 MiB GO alloc/idle/total, 90 MiB/140 MiB CGO alloc/total, 1723.9 CGO/sec, 132.1/11.0 %(u/s)time, 0.1 %gc (3x), 3.5 MiB/3.5 MiB (r/w)net
I190827 19:40:49.642842 141095 storage/replica_command.go:293  [n1,s1,r22/1:/{Table/53-Max}] initiating a split of this range at key /Table/55/1 [r23] (manual)
I190827 19:40:49.879274 141027 storage/replica_command.go:293  [n1,s1,r22/1:/Table/5{3-5/1}] initiating a split of this range at key /Table/54/1 [r24] (manual)
I190827 19:40:50.325280 138359 server/status/runtime.go:498  [n3] runtime stats: 2.0 GiB RSS, 719 goroutines, 149 MiB/133 MiB/330 MiB GO alloc/idle/total, 91 MiB/140 MiB CGO alloc/total, 1719.0 CGO/sec, 132.6/11.1 %(u/s)time, 0.1 %gc (3x), 3.4 MiB/3.4 MiB (r/w)net
I190827 19:40:50.767908 141072 storage/replica_command.go:293  [n1,split,s1,r22/1:/Table/5{3-4/1}] initiating a split of this range at key /Table/54 [r25] (zone config)
I190827 19:40:50.872898 141298 storage/replica_command.go:293  [n1,split,s1,r24/1:/Table/5{4/1-5/1}] initiating a split of this range at key /Table/55 [r26] (zone config)



TestImportMysql/second_from_multi
...792 +0000 UTC
I190827 19:41:07.406762 139891 sql/event_log.go:130  [n1,client=127.0.0.1:49848,user=root] Event: "drop_table", target: 58, info: {TableName:foo.public.simple Statement:DROP TABLE IF EXISTS simple, second, third, everything CASCADE User:root CascadeDroppedViews:[]}
I190827 19:41:07.640766 139891 sql/lease.go:329  [n1,client=127.0.0.1:49848,user=root,scExec] waiting for 1 leases to expire: desc=[{simple_auto_inc 57 2}]
I190827 19:41:07.660121 144066 sql/lease.go:923  new lease: 57("simple_auto_inc") ver=3:1566935155.071241731,0, refcount=0
I190827 19:41:07.827417 144200 storage/replica_command.go:608  [n1,merge,s1,r33/1:/Table/58{-/1}] initiating a merge of r31:/{Table/58/1-Max} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=17] into this range (lhs+rhs has (size=0 B+18 KiB qps=0.00+3.82 --> 3.82qps) below threshold (size=18 KiB, qps=3.82))
I190827 19:41:08.037635 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root,scExec] publish: descID=58 (simple) version=4 mtime=2019-08-27 19:41:08.035762312 +0000 UTC
I190827 19:41:08.197038 137134 storage/store.go:2593  [n1,s1,r33/1:/Table/58{-/1}] removing replica r31/1
I190827 19:41:08.203278 138264 storage/store.go:2593  [n3,s3,r33/3:/Table/58{-/1}] removing replica r31/3
I190827 19:41:08.205998 137981 storage/store.go:2593  [n2,s2,r33/2:/Table/58{-/1}] removing replica r31/2
I190827 19:41:09.067687 138030 server/status/runtime.go:498  [n2] runtime stats: 2.0 GiB RSS, 735 goroutines, 207 MiB/82 MiB/330 MiB GO alloc/idle/total, 98 MiB/146 MiB CGO alloc/total, 1056.3 CGO/sec, 124.0/11.9 %(u/s)time, 0.1 %gc (2x), 2.4 MiB/2.4 MiB (r/w)net
I190827 19:41:09.149573 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root] publish: descID=57 (simple_auto_inc) version=4 mtime=2019-08-27 19:41:09.012051386 +0000 UTC
I190827 19:41:09.340781 139891 sql/event_log.go:130  [n1,client=127.0.0.1:49848,user=root] Event: "drop_sequence", target: 57, info: {SequenceName:foo.public.simple_auto_inc Statement:DROP SEQUENCE IF EXISTS simple_auto_inc, third_auto_inc User:root}
I190827 19:41:09.524686 139891 sql/lease.go:329  [n1,client=127.0.0.1:49848,user=root,scExec] waiting for 1 leases to expire: desc=[{simple_auto_inc 57 3}]
I190827 19:41:09.888724 144545 storage/replica_command.go:608  [n1,merge,s1,r34/1:/Table/57{-/1}] initiating a merge of r32:/Table/5{7/1-8} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=19] into this range (lhs+rhs has (size=0 B+41 B qps=0.00+2.53 --> 2.53qps) below threshold (size=41 B, qps=2.53))
I190827 19:41:10.056157 139891 sql/sqlbase/structured.go:1511  [n1,client=127.0.0.1:49848,user=root,scExec] publish: descID=57 (simple_auto_inc) version=5 mtime=2019-08-27 19:41:10.053991521 +0000 UTC
I190827 19:41:10.385558 138359 server/status/runtime.go:498  [n3] runtime stats: 2.0 GiB RSS, 780 goroutines, 132 MiB/150 MiB/330 MiB GO alloc/idle/total, 100 MiB/148 MiB CGO alloc/total, 1004.5 CGO/sec, 125.6/11.9 %(u/s)time, 0.4 %gc (3x), 2.4 MiB/2.4 MiB (r/w)net
I190827 19:41:10.434460 137171 storage/store.go:2593  [n1,s1,r34/1:/Table/57{-/1}] removing replica r32/1
I190827 19:41:10.446064 137946 storage/store.go:2593  [n2,s2,r34/2:/Table/57{-/1}] removing replica r32/2
I190827 19:41:10.466151 138303 storage/store.go:2593  [n3,s3,r34/3:/Table/57{-/1}] removing replica r32/3
I190827 19:41:11.495192 145124 storage/replica_command.go:293  [n1,s1,r33/1:/{Table/58-Max}] initiating a split of this range at key /Table/59/1 [r35] (manual)
I190827 19:41:12.016478 145124 storage/replica_command.go:293  [n1,s1,r35/1:/{Table/59/1-Max}] initiating a split of this range at key /Table/59/2 [r36] (manual)
I190827 19:41:12.553268 145124 storage/replica_command.go:293  [n1,s1,r36/1:/{Table/59/2-Max}] initiating a split of this range at key /Table/59/3 [r37] (manual)
I190827 19:41:14.443006 145394 storage/replica_command.go:293  [n1,split,s1,r33/1:/Table/5{8-9/1}] initiating a split of this range at key /Table/59 [r38] (zone config)



TestImportMysql/all_from_multi_gzip
...utine 154521 [chan receive, 35 minutes]:
github.com/cockroachdb/cockroach/pkg/sql/distsqlrun.(*RowChannel).Next(0xc006cb9e00, 0x66a8820, 0xc00a22e9e0, 0xc006cb9e40, 0xc00621cda8)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsqlrun/base.go:438 +0x6c
github.com/cockroachdb/cockroach/pkg/sql/distsqlrun.(*noopProcessor).Next(0xc008dd3200, 0x4, 0x66a8820, 0xc006cb9e40, 0x66a8820)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsqlrun/noop.go:66 +0x86
github.com/cockroachdb/cockroach/pkg/sql/distsqlrun.Run(0x66a8820, 0xc006cb9e40, 0x66cf460, 0xc008dd3200, 0x66824e0, 0xc0089bdc00)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsqlrun/base.go:171 +0x43
github.com/cockroachdb/cockroach/pkg/sql/distsqlrun.(*ProcessorBase).Run(0xc008dd3200, 0x66a8820, 0xc006cb9e40)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsqlrun/processors.go:793 +0x114
github.com/cockroachdb/cockroach/pkg/sql/distsqlrun.(*Flow).Run(0xc009bc67e0, 0x66a88e0, 0xc003d4aa80, 0x57b3198, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsqlrun/flow.go:584 +0x40a
github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).Run(0xc008da1cc0, 0xc0023fa720, 0xc006d685a0, 0xc00cc85d58, 0xc0084402c0, 0xc007b82430, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:334 +0x585
github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).PlanAndRun(0xc008da1cc0, 0x66a88e0, 0xc0061b8960, 0xc007b82430, 0xc0023fa720, 0xc006d685a0, 0x66aa3a0, 0xc009bc6000, 0xc0084402c0, 0xc007b821e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:959 +0x272
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execWithDistSQLEngine(0xc007b82000, 0x66a88e0, 0xc0061b8960, 0xc007b82360, 0x3, 0x7f61e9875b68, 0xc00856dce0, 0xc00cc86101, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:816 +0x63b
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine(0xc007b82000, 0x66a88e0, 0xc0061b8960, 0xc007b82360, 0x7f61e9875b68, 0xc00856dce0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:708 +0xc0b
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState(0xc007b82000, 0x66a88e0, 0xc0061b8960, 0x66c8360, 0xc006cb8b40, 0x570b4b6, 0x51, 0x3, 0x1, 0xc005ea7240, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:417 +0xc86
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt(0xc007b82000, 0x66a88e0, 0xc0061b8960, 0x66c8360, 0xc006cb8b40, 0x570b4b6, 0x51, 0x3, 0x1, 0xc005ea7240, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:98 +0x6d7
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd(0xc007b82000, 0x66a8820, 0xc005ea7340, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1300 +0x1428
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc007b82000, 0x66a88e0, 0xc00668b410, 0xc0075ced20, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1167 +0x3e8
github.com/cockroachdb/cockroach/pkg/sql.(*internalExecutorImpl).initConnEx.func1(0xc007b82000, 0x66a88e0, 0xc00668b410, 0xc0029c4960, 0xc00783cd20, 0x0, 0xc00032a784)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:201 +0x8f
created by github.com/cockroachdb/cockroach/pkg/sql.(*internalExecutorImpl).initConnEx
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:200 +0xa2e

goroutine 138149 [select, 36 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x66a88e0, 0xc003dc4840)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:94 +0x1be
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc005f13610, 0xc0099caf00, 0xc005f135d0)

goroutine 137128 [sync.Cond.Wait]:


***** Text was not loaded fully because its' size exceeds 2 MB, see full log for the whole text *****





```

Please assign, take a look and update the issue accordingly.

--- a/pkg/ccl/importccl/import_stmt_test.go
+++ b/pkg/ccl/importccl/import_stmt_test.go
@@ -5237,8 +5237,6 @@ func TestImportMysql(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	skip.WithIssue(t, 40263)
-
 	const (
 		nodes = 3
 	)
@@ -5339,7 +5337,8 @@ func TestImportMysql(t *testing.T) {
 					if got, expected := res[i][0], fmt.Sprintf("%v", everythingTestRows[i].i); got != expected {
 						t.Fatalf("expected %s got %s", expected, got)
 					}
-					if got, expected := res[i][1], everythingTestRows[i].c; got != expected {
+					if got, expected := res[i][1],
+						fmt.Sprintf("%v", everythingTestRows[i].c); got != expected {
 						t.Fatalf("expected %s got %s", expected, got)
 					}
 					if got, expected := res[i][2], fmt.Sprintf("%v", everythingTestRows[i].iw); got != expected {

--- a/pkg/ccl/importccl/mysql_testdata_helpers_test.go
+++ b/pkg/ccl/importccl/mysql_testdata_helpers_test.go
@@ -105,8 +105,12 @@ type everythingTestRow struct {
 
 var everythingTestRows = func() []everythingTestRow {
 	return []everythingTestRow{
-		{1, "Small", "c", []byte("bin"), timeutil.Unix(946684800, 0), 1, -2, -1.5, "-12.345", `{"a": "b", "c": {"d": ["e", 11, null]}}`},
-		{2, "Large", "c2", []byte("bin2"), timeutil.Unix(946684800, 0), 3525343334, 3, 1.2, "12.345", `{}`},
+		{1, "Small", string([]byte{99, 32, 32, 32, 32, 32, 32, 32, 32, 32}), []byte("bin"),
+			timeutil.Unix(946684800, 0), 1, -2, -1.5, "-12.345",
+			`{"a": "b", "c": {"d": ["e", 11, null]}}`},
+		{2, "Large", string([]byte{99, 50, 32, 32, 32, 32, 32, 32, 32, 32}), []byte("bin2"),
+			timeutil.Unix(946684800, 0), 3525343334, 3,
+			1.2, "12.345", `{}`},
 	}
 }()
 

